support/index.html

```diff
@@ -139,22 +139,22 @@ <h1 class="navbar-brand">
             state: 'bar'
           }
         },
-        additionalSignUpFields: [
-          {
-            name: 'name',
-            placeholder: 'name',
-            validator: function() {
-              return true;
-            }
-          },
-          {
-            name: 'other_name',
-            placeholder: 'other name',
-            validator: function() {
-              return true;
-            }
-          }
-        ]
+        // additionalSignUpFields: [
```

Comment:
why not just remove instead of commenting? git history can keep track of removals.

Reply:
This was mistakenly committed. Will remove.